dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22ComponentRendererTests.java

```diff
@@ -288,4 +289,28 @@ public void testJsf599() throws Exception {
             assertTrue(page.asText().contains("I'm a place!"));
         }
     }
+
+        /*
+     *  Two items are tested. 
+     *  1) Default tag name is assigned when one is not specified on the @FacesComonent annotation.
+     *  2) Verifies a customer renderer is used when specified. This was added because we don't have any 
+     *  exisiting tests for the @FacesRenderer annotation.   The component doesn't render the element, but
+     *  the renderer. 
```

Comment:
`but the renderer` does what?

Reply:
Modified to say: ` The component class doesn't render the element, but
     *  the renderer class does. `